{{cookiecutter.project_slug}}/HELP.md

```diff
@@ -47,132 +47,7 @@ make jupyter
 ### Kill Jupyter
 
 ```shell 
-make kill-jupyter
+neuro-flow kill jupyter
 ```
 
 * The job with Jupyter Notebooks is terminated. The notebooks are saved on the platform storage. You may run `make download-notebooks` to download them to the local `notebooks/` directory.
-
-### Help
-
-```shell 
-make help
-```
-
-* The list of all available template commands is printed.
-
-## Data
-
-### Uploading to the Storage via Web UI
-
-On local machine, run `make filebrowser` and open the job's URL on your mobile device or desktop.
-Through a simple file explorer interface, you can upload test images and perform file operations.
-
-### Uploading to the Storage via CLI
-
-On local machine, run `make upload-data`. This command pushes local files stored in `./data`
-into `storage:{{ cookiecutter.project_slug }}/data` mounted to your development environment's `/project/data`.
-
-### Uploading data to the Job from Google Cloud Storage
-
-Google Cloud SDK is pre-installed on all jobs produced from the Base Image.
-
-Please refer to the [documentation](https://docs.neu.ro/toolbox/accessing-object-storage-in-gcp) explaining how to start using GCP with the template.
-
-### Uploading data to the Job from AWS S3
-
-AWS CLI is pre-installed on all jobs produced from the Base Image.
-
-Please refer to the [documentation](https://docs.neu.ro/toolbox/accessing-object-storage-in-aws) explaining how to start using AWS with the template.
-
-## Customization
```

Comment:
Maybe let's leave this section, but just changing pointers onto `/{{ cookiecutter.project_slug }}/.neuro/live.yml` file?
Since this functionality (e.g. python debugging, updating data manually) is preserved (except Weights & Biases) and may be helpful.

Reply:
[resolving](https://github.com/neuro-inc/cookiecutter-neuro-project/pull/419#discussion_r498289193)